tests/functional/test_misc.py

```diff
@@ -1350,7 +1350,7 @@ def test_create_and_combine_stream(self):
         self.session = Session(MIN_BLOB_DATA_PAYMENT_RATE, db_dir=db_dir, lbryid="abcd",
                                    peer_finder=peer_finder, hash_announcer=hash_announcer,
                                    blob_dir=blob_dir, peer_port=5553,
-                                   use_upnp=False, rate_limiter=rate_limiter, wallet=wallet)
+                                   use_upnp=False, rate_limiter=rate_limiter, wallet=wallet, blob_tracker_class=DummyBlobAvailabilityTracker)
```

Comment:
I'd suggest changing your pycharm settings to 100 characters:
http://stackoverflow.com/questions/17319422/where-to-set-maximum-line-length-in-pycharm.

In my experience 100 characters is about as wide as you can get and still allow for side-by-side editting / diff viewing on a laptop.